chia/rpc/wallet_rpc_api.py

```diff
@@ -636,7 +636,7 @@ async def create_new_wallet(self, request: Dict) -> EndpointResult:
         if request["wallet_type"] == "cat_wallet":
             # If not provided, the name will be autogenerated based on the tail hash.
             name = request.get("name", None)
-            if request["mode"] == "new":
+            if request["mode"] == "new test":
```

Comment:
My recommendation would be to avoid overloading the `"mode"` by defining a new value, and instead, add a new `"test": true` key/value. Then the `request["mode"] == "new"` branch can remain the same but be dependent on the test key.

If that's not workable, I would at least change the mode to "new_test" instead of starting a pattern of including spaces in param values.